src/main/java/uk/gov/justice/laa/portal/landingpage/service/CreateUserNotificationService.java

```diff
@@ -40,10 +39,9 @@ public void notifyCreateUser(String username, String email, String password, Str
         }
     }
 
-    public Map<String, String> addProperties(String username, String password) {
+    public Map<String, String> addProperties(String username) {
 
         customProps.put(USER_NAME, username);
-        customProps.put(PASSWORD, password);
```

Comment:
you also need to update the email template at notify site. I could see the ((password)) is still in welcome template